cli/src/samples.rs

```diff
@@ -37,61 +46,58 @@ pub fn get_trusted_peers(public_key: Option<&PublicKey>) -> HashSet<PeerId> {
     .collect();
     if let Some(pubkey) = public_key {
         trusted_peers.insert(PeerId {
-            address: DEFAULT_TORII_P2P_ADDR.clone(),
+            address: DEFAULT_P2P_ADDR.clone(),
             public_key: pubkey.clone(),
         });
     }
     trusted_peers
 }
 
 #[allow(clippy::implicit_hasher)]
-/// Get a sample Iroha configuration proxy. Trusted peers must be
+/// Get a sample Iroha configuration on user-layer level. Trusted peers must be
```

Comment:
What does "on user-layer level" mean?

Reply:
There are now two layers of the configuration:

- User: has a structure and types to provide convenient format for users to set in file/env
- Parsed (or actual): stronger types, validated content, in some places even different structure (e.g. split into internal modules)

Reply:
Extended documentation of `iroha_config` and `iroha_client::config` modules with this description